test_builder/test_builder

```diff
@@ -708,7 +708,21 @@ function grpc_protocol_driver_setup() {
     server_settings {
       name: 'server_type'
       string_value: '${SERVER_TYPE}'
-    }
+    }"
+
+  if [[ $SERVER_TYPE != "inline" ]];
+  then
+    set_parameter_default threadpool_size "4" "0, 1, 2, ..."
```

Comment:
How does a user allow the default of NumCpus to be used?

Reply:
I think there was some miscommunication. I meant that the default value passed to GetNamedServerSettings() for threadpool_size was NumCPUs(). 

But anyways, modified test_builder to accept the value of -1 in order not configure threadpool_size in protofile so that the test servers have NumCPU() threads.